api/handlers.go

```diff
@@ -493,6 +494,15 @@ func (api *API) userPUT(w http.ResponseWriter, req *http.Request, _ httprouter.P
 		return
 	}
 
+	if payload.Password != "" {
```

Comment:
I wish I could comment outside the diff. 

But if the password isn't passed in, or if any individual fields isn't passed in, is this line then overwritten with empty fields https://github.com/SkynetLabs/skynet-accounts/pull/97/files#diff-855b4003387cd9e2b8e2c353c747fd1817eccf1d223b4066f0d7a1db0b4b4b4eR561

Reply:
I'm not sure which line you have in mind but nothing is overwritten unless explicitly passed.
What we do here is the following:
 - fetch the user from the DB
 - update the fields of the struct that we have validated values for
 - save the struct back to the DB with a `replace` statement

Any changes that we explicitly do here are preserved in the DB but no fields get reset to their default values.